Haulage/HaulageTests/DatabaseTests.cs

```diff
@@ -0,0 +1,108 @@
+//namespace Haulage.HaulageTests.Console;
+
+//using System;
+//using System.Data.SQLite;
```

Comment:
commented code either remove or uncomment

Reply:
Thank you, htis was missed and has now been removed.